### ISSUE TITLE
Datepicker, Multiselect, Message, Toast: Missing aria-labels

### ISSUE THREAD

**Issue by TaneliTuomola** [primefaces/primeng#17449]
### Describe the bug

- Datepicker: Missing aria-label on prev and next buttons
- MultiSelect: Missing aria-label on checkboxes
- Message: close button missing aria-label
- Toast: close button missing aria-label

### Pull Request Link

https://github.com/primefaces/primeng/pull/17448

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/

### Environment

https://primeng.org/

### Angular version

19

### PrimeNG version

v19

### Steps to reproduce the behavior

1. primeNg.org 2. Open component doc 3. Inspect elements

### Expected behavior

Aria-labels fixed
